README.md

```diff
@@ -67,6 +67,8 @@ Then source this file before starting rails:
 $ . ~/s3-envvars.sh; bundle exec rails s -p 3000
 ```
 
+By default `storage.yml` is also configured to use local storage instead of connecting to AWS. Make sure you update your local copy as indicated inside the file if you want to use AWS.
```

Comment:
This is not technically needed but I keep running into issues because my local env is not uploading all the way to AWS so I wanted to document it.